app/components/notification_settings_email_android/notification_settings_email_android.js

```diff
@@ -0,0 +1,298 @@
+
+import FormattedText from 'app/components/formatted_text';
+import RadioButtonGroup from 'app/components/radio_button';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+export default class NotificationSettingsEmailAndroid extends PureComponent {
+    static propTypes = {
+        actions: PropTypes.shape({
+            savePreferences: PropTypes.func.isRequired,
+        }),
+        currentUserId: PropTypes.string.isRequired,
+        emailInterval: PropTypes.string.isRequired,
+        enableEmailBatching: PropTypes.bool.isRequired,
+        onClose: PropTypes.func.isRequired,
+        sendEmailNotifications: PropTypes.bool.isRequired,
+        siteName: PropTypes.string,
+        theme: PropTypes.object.isRequired,
+        visible: PropTypes.bool.isRequired,
+    };
+
+    static contextTypes = {
+        intl: intlShape,
+    };
+
+    constructor(props) {
+        super(props);
+
+        const {
+            emailInterval,
+            enableEmailBatching,
+            sendEmailNotifications,
+        } = props;
+
+        this.state = {
+            interval: getEmailInterval(
+                sendEmailNotifications,
+                enableEmailBatching,
+                parseInt(emailInterval, 10),
+            ).toString(),
+        };
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (
+            this.props.sendEmailNotifications !== nextProps.sendEmailNotifications ||
+            this.props.enableEmailBatching !== nextProps.enableEmailBatching ||
+            this.props.emailInterval !== nextProps.emailInterval
+        ) {
+            this.setState({
+                interval: getEmailInterval(
+                    nextProps.sendEmailNotifications,
+                    nextProps.enableEmailBatching,
+                    parseInt(nextProps.emailInterval, 10),
+                ).toString(),
+            });
+        }
+    }
+
+    setEmailNotifications = (interval) => {
+        const {sendEmailNotifications} = this.props;
+
+        let email = 'false';
+        if (sendEmailNotifications && interval !== Preferences.INTERVAL_NEVER.toString()) {
+            email = 'true';
+        }
+
+        this.setState({
+            email,
+            interval,
+        });
+    };
+
+    handleClose = () => {
+        this.props.onClose();
+    }
+
+    handleSaveEmailNotification = () => {
+        const {currentUserId} = this.props;
+        const {email, interval} = this.state;
+
+        const emailNotify = {category: Preferences.CATEGORY_NOTIFICATIONS, user_id: currentUserId, name: 'email', value: email};
+        const emailInterval = {category: Preferences.CATEGORY_NOTIFICATIONS, user_id: currentUserId, name: Preferences.EMAIL_INTERVAL, value: interval};
+        this.props.actions.savePreferences(currentUserId, [emailNotify, emailInterval]);
+        this.props.onClose();
+    };
+
+    render() {
+        if (Platform.OS === 'ios') {
```

Comment:
You can create platform specific files. instead of `app/components/notification_settings_email_android/notification_settings_email_android.js` you can do `app/components/notification_settings_email/notification_settings_email.android.js` and then you don't need to check for Platform.

Also in this case you could render null if visible if false and avoid the rest of the code to be ran